### ISSUE TITLE
extension fails if you have showSignature enabled in git settings

### ISSUE THREAD

**Issue by k-yle** [mhutchie/vscode-git-graph#452]
**Describe the Bug**

In your `~/.gitconfig` file, add this:

```ini
[log]
  showSignature = true
```

and now the extension doesn't work at all, it just renders a blank screen (see screenshot below)

**Steps to Reproduce**
Steps to reproduce the behaviour:
1. open `~/.gitconfig` in an editor
2. add this:
      ```ini
      [log]
         showSignature = true
   ```

3. try to open the extension
4. note how that it's blank (see screenshot below)

**Expected Behaviour**

it should work 

**Environment**
 - Git Graph Extension Version: 1.28.0
 - Visual Studio Code Version: 1.52.1
 - Operating System: Windows 10

**Screenshots (optional)**

![image](https://user-images.githubusercontent.com/16009897/104981422-74a77a00-5a6d-11eb-82ea-534b7cb78165.png)

**Additional Context (optional)**

when you enable ` showSignature = true` it makes the git log really ugly because it prints the GPG signature details for each signed comment 

![image](https://user-images.githubusercontent.com/16009897/104981642-dff14c00-5a6d-11eb-9486-3ca186c70c33.png)

presumably this extension should provide a flag to git log to override the user settings for `showSignature`

**Comment by mhutchie** [comment]
Hi @k-yle,

When you have the Git configuration variable `log.showSignature` set to `true`, it instructs Git to always include the signature information in the `git log` output, regardless of whether it is being run in a terminal or programmatically by some Git UI. When Git Graph runs `git log` to obtain the commit information for the repository, it specifies the format (via `--format=...`) that Git should use when returning the commits so they can be accurately parsed by the extension. Unfortunately if `log.showSignature = true` is set, when the extension runs `git log` Git doesn't respond with the requested format, and instead includes un-parseable terminal specific mark-up in the response. Due to the nature of this incorrect response formatting, the extension determines each commit to be invalid and ignores them (as it does this for all commits, the UI display it as no commits).

There isn't a specific argument for `git log` to disable including the signature information in the response, however I can override the configuration variable when the extension runs the specific command via `git -c log.showSignature=false log ...` to achieve the desired affect, and make the response parseable.

I'll include this fix in a beta release overnight for you to use.

**Comment by mhutchie** [comment]
Thanks again for making me aware of this!

This fix be available in v1.29.0.

If you'd like to use it before the next release, you can download [v1.29.0-beta.5](https://github.com/mhutchie/vscode-git-graph/releases/tag/v1.29.0-beta.5), and install it following the instructions provided [here](https://github.com/mhutchie/vscode-git-graph/wiki/Downloading-&-Installing-Releases#manual-installation).

**Comment by k-yle** [comment]
thanks for the super fast fix @mhutchie !
